<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbView</class>
 <widget class="QDialog" name="DbView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据报表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>成绩记录</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>游戏记录：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tvGames"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>各轮次数据：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tvRounds"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>得分明细：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tvShootings"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>营业资料</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>收费记录：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tvPayment"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>汇总数据，不再关注</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>520</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>导出数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDbReset">
        <property name="text">
         <string>清空数据</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
